bitbot/util.go

```diff
@@ -37,6 +37,29 @@ type ACL struct {
 	Rejected []string
 }
 
+//contains returns a bool if the string is found in the slice of strings
+func contains(list []string, item string) (int, bool) {
+	for i, val := range list {
+		if item == val {
+			return i, true
+		}
+	}
+	return -1, false
```

Comment:
If there is no match, the integer returned should be a `zero` value for that type (`0` in this case). Ultimately it's up to the consumer of the function to check the bool and discard the index value if `false`

https://tour.golang.org/basics/12